### PR TITLE
create a least privilege policy that looks for a role using wildcards

### PR DESCRIPTION
**Pull request by gparvin**
This is a sample policy I'm creating as part of the blog entry I'm writing for using the policy collection repo.  Hopefully this looks like an interesting policy too.  It does help provide some least privilege coverage which is a control that is not currently being analyzed by any community policies.

Feedback welcome!

### CODE REVIEW

**Review by ycao56 — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]
made a very small change, looks good to me

File: community/AC-Access-Control/policy-roles-no-wildcards.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-disallowed-roles
+  namespace: default
```
Comment (ycao56) [inline community/AC-Access-Control/policy-roles-no-wildcards.yaml]: remove ns

File: community/AC-Access-Control/policy-roles-no-wildcards.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    resources:
+                      - '*'
+                    verbs:
+                      - '*'
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-policy-disallowed-roles
+  namespace: default
```
Comment (ycao56) [inline community/AC-Access-Control/policy-roles-no-wildcards.yaml]: remove ns

File: community/AC-Access-Control/policy-roles-no-wildcards.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+subjects:
+- name: policy-disallowed-roles
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-policy-disallowed-roles
+  namespace: default
```
Comment (ycao56) [inline community/AC-Access-Control/policy-roles-no-wildcards.yaml]: remove ns

**Pull request merged**